tests/Feature/BerandaPublik/TodayCardLecturingScheduleTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Tests\Feature\LecturingSchedule;
```

Comment:
Mas @andriawan ini kalau boleh, struktur folder dan nama classnya kita ganti begini:

tests/Feature/Livewire/PublicHome/TodayLecturingSchedulesTest.php

```suggestion
namespace Tests\Feature\Livewire\PublicHome;
```

Reply:
setuju mas @nafiesl, untuk penamaan memang perlu diskusi.